plugins/modules/aci_l4l7_device_selection_interface_context.py

```diff
@@ -0,0 +1,414 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# Copyright: (c) 2025, Shreyas Srish (@shrsr)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_device_selection_interface_context
+short_description: Manage L4-L7 Device Selection Policy Logical Interface Contexts (vns:LIfCtx)
+description:
+- Manage L4-L7 Device Selection Policy Logical Interface Contexts
+options:
```

Comment:
I see missing option in the model, did you check ui and model? permitHandoff, ruleType